.github/ISSUE_TEMPLATE/1_general_bug_report.yaml

```diff
@@ -0,0 +1,101 @@
+name: Report a bug
+description: Errors and regression reports with complete reproducing test cases and/or stack traces.
+labels: ["status:needs-triage", "bug"]
+body:
+  - type: markdown
+    attributes:
+      value: >
+        Many bug reports are often **USAGE QUESTIONS, NOT BUGS**. If you do not have a strong understanding 
+        of the interface you are reporting a bug for, please head to our [Community Slack](https://www.prefect.io/slack/)
+        or [Discourse](https://discourse.prefect.io/) and ask there first. You are likely to get a response
+        faster and learn more about the feature you're working with. If the issue is determined to be a bug,
+        we will open an issue here.
+  
+  - type: markdown
+    attributes:
+      value: >
+        If you are new to Prefect bug reports, please review our many examples of [well written bug reports](https://github.com/PrefectHQ/prefect/issues?q=is%3Aissue+label%3A%22great+writeup%22). 
+        Each of these reports include the following features:
+
+        1. A **succinct description of the problem** - typically a line or two at most
+
+        2. A **succinct code example which reproduces the problem**, otherwise known as a [Minimal, Complete, and Verifiable](https://stackoverflow.com/help/mcve) example.
```

Comment:
```suggestion
        2. A **succinct code example which reproduces the problem**, otherwise known as a [minimal, complete, and verifiable](https://stackoverflow.com/help/mcve) example.
```